datafusion/src/physical_plan/expressions/mod.rs

```diff
@@ -61,7 +61,7 @@ pub use is_not_null::{is_not_null, IsNotNullExpr};
 pub use is_null::{is_null, IsNullExpr};
 pub use lead_lag::{lag, lead};
 pub use literal::{lit, Literal};
-pub use min_max::{Max, Min};
+pub use min_max::{Max, MaxAccumulator, Min, MinAccumulator};
```

Comment:
what would you think about making this `pub (crate) use min_max::{MaxAccumulator, MinAccumulator}`  rather than `pub use ...`? 

It seems fine to use them within DataFusion but exposing them externally might lead to people relying on them accidentally

Reply:
I did not consider this situation, it has been modified.